wrappers/bedtools/bamtofastq/wrapper.py

```diff
@@ -0,0 +1,16 @@
+from snakemake.shell import shell
+
+log = snakemake.log_fmt_shell(stdout=True, stderr=True)
+
+outdir = snakemake.output.fastq1.split('/')[0]
+
+shell(
+
+    "(samtools sort -n  "
+    "-@ {snakemake.threads} "
+    "-T {outdir}/{snakemake.wildcards.sample} "
```

Comment:
It might be better for the rule to supply the full path to a temporary working directory rather than have this logic hidden in the wrapper.

Reply:
yeah agreed, this is a bit awkward- i initially specified an output directory in the rule but snakemake complained about wildcards. should be able to find a workaround!